### ISSUE TITLE
Unclear details around Nodes and `NetworkUnavailable: false`

### ISSUE THREAD

**Issue by jayunit100** [kubernetes/website#42917]
It appears https://kubernetes.io/docs/concepts/architecture/nodes/#condition (sorry i meant https://kubernetes.io/docs/reference/node/node-status/#condition) 

is missing any mention of how the `NetworkUnavailable` condition = false should be interpretted. 

IF this is an "official" API that we rely on then i think its ok to rely on it in the e2e tests.  but if its not an officially supported standard, i think the e2e tests shouldnt force this field to be set to "False" in order to run. 

i.e. 			
```
networkReady := isConditionUnset(&node, v1.NodeNetworkUnavailable) || IsConditionSetAsExpected(&node, v1.NodeNetworkUnavailable, false)
```
is relying on the definition of NodeNetworkUnavialable:false to mean something like "This node should not be thought of as , in any way , testable / usable for running pods".

However, this condition is just a vanity metric.  You can easily have a node that does run pods, which simply doesnt have this attached metadata, and as a metric, its managed by third party libraries / cnis /etc... so K8s itself i dont think should really rely on it... unless we document it as w/ a strict definition. 

Feel free to close if im just missing the docs and theres a better place to look this up . I do see its in the v1.Types ... but i think at the very least, we need it documented on https://kubernetes.io/docs/concepts/architecture/nodes/#condition if were going to keep relying on it.

**Comment by sftim** [comment]
I think the page to fix is https://kubernetes.io/docs/reference/node/node-status/#condition

All https://k8s.io/docs/concepts/architecture/nodes/#node-status needs to do is clearly signpost people to the reference page.

**Comment by jayunit100** [comment]
yeah agree that would do it, we also need to disambiguate the different types of things that may toggle the value of this setting to since its so far out of bounds of what k/k/core does

**Comment by jayunit100** [comment]
xref https://github.com/kubernetes/kubernetes/issues/120486 bc its a broader issue of disambiguation, talked w @aojea

**Comment by sftim** [comment]
/retitle Unclear details around Nodes and `NetworkUnavailable: false`
